lib/watchable-message-store/WatchableMessageStore.d.ts

```diff
@@ -46,6 +46,7 @@ export declare class WatchableMessageStore {
     private bcc?;
     private dbName;
     private objStoreName;
+    private _lastCursorId;
```

Comment:
No need for _, surely? Or are you concerned about the JS side?

Reply:
Matter of habit. Fixed.